website/src/components/CustomerTestimonials/index.tsx

```diff
@@ -69,19 +72,28 @@ const TestimonialBox = ({
   authorName,
   companyName,
   authorTitle,
+  fontSize,
 }: TestimonialBoxProps) => {
   return (
-    <div className="first:sm:ml-48 last:sm:mr-48 shrink-0 px-8 md:px-12 py-8 md:py-12 bg-[#1B1B1D] snap-center flex flex-col rounded-2xl justify-between w-fit max-w-[540px] h-84">
-      <p className="text-md md:text-lg tracking-wide font-light mb-2 md:mb-6 break-keep italic text-neutral-50">
+    <div className="shrink-0 p-6 md:p-8 bg-[#1B1B1D] flex flex-col rounded-2xl justify-between w-fit lg:max-w-[320px] xl:max-w-[340px] lg:min-h-[320px] h-fit">
+      <FaQuoteLeft
+        width={32}
+        height={32}
+        className="absolute text-white/10 -translate-y-1/3 -z-1 w-10 h-10 sm:h-12 md:h-12 md:w-12"
```

Comment:
```suggestion
        className="absolute text-white/10 -translate-y-1/3 -z-1 w-10 h-10 sm:h-12 md:h-12 md:w-12"
```

I think you just need the className